src/index.js

```diff
@@ -200,9 +235,13 @@ async function sendWebHook(content, name, version, reporter, exception, dhash, e
   };
 
   if (reporter && !checkForbidden(reporter)) {
-    body.embeds[0].author = {
-      "name": reporter
-    };
+    body.embeds[0].author["name"] = reporter;
```

Comment:
This change will result in an error being thrown because `body.embeds[0].author` is undefined. Easiest fix would seem to be, in the above `let body = ` block, adding something like this to the embed:
```javascript
"author": {
    "name": "Anonymous Reporter"
}
```

That will ensure `body.embeds[0].author` is always defined and you can then override its `name` and `icon_url` as desired safely.